### PR TITLE
Implement proper handling of OTF videos.

### PR DESCRIPTION
**Pull request by FireMasterK**

### CODE REVIEW

File: app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt [hunk 1]
```diff
@@ -1193,13 +1193,18 @@ class PlayerFragment : BaseFragment(), OnlinePlayerOptions {
         if (!PreferenceHelper.getBoolean(PreferenceKeys.USE_HLS_OVER_DASH, false) &&
             streams.videoStreams.orEmpty().isNotEmpty()
         ) {
-            val manifest = DashHelper.createManifest(streams)
+            val uri = let {
```
Comment (Bnyro) [inline app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt:1196]: If the goal was to use dash if available that actually won't work like that :p

Comment (Bnyro) [reply app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt:1196]: let will always take the last statement, it needs to be used differently :)

Comment (Bnyro) [reply app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt:1196]: Or maybe I've just no idea how it works ...

Comment (FireMasterK) [reply app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt:1196]: Well, it was actually:
```
if (dash != null) {
    return dash
}
```

IntelliJ / Android studio suggested that, and it worked like what I wrote!

**Pull request merged**